app/views/layouts/application.html.erb

```diff
@@ -1,23 +1,24 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>MediSafe - Everyone must to be safe!</title>
-    <%= csrf_meta_tags %>
-    <%= csp_meta_tag %>
+<head>
```

Comment:
<head> should be indented.

Reply:
Done
It's a bunch of configs for meta tags, copy/past most part.